### ISSUE TITLE
[Bug Report][3.9.5] V-Select Options Cannot Be Selected When Using Windows Narrator Screenreader

### ISSUE THREAD

**Issue by maxg-uiuc** [vuetifyjs/vuetify#21935]
### Environment
**Vuetify Version:** 3.9.5
**Vue Version:** 3.5.6
**Browsers:** Google Chrome
**OS:** Windows

### Steps to reproduce
Open Windows Narrator. Tab into the V-Select. Attempt to select an option using the arrow keys.

### Expected Behavior
You will be able to navigate up and down the list, and select an option.

### Actual Behavior
The Narrator gets stuck immediately when you try to navigate down, and you cannot tab onto any items.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNptkk1Lw0AQhv/KsAc/ICYHqWCJgr0olIJYwUPTw5hOdMlmu+xOakPpf3eTGGMTL7szz7vMvDPs6iCcTaMHY8JdSWIqYqbCKGS6TzRAvLtCY5qwSdKtZpSa7A9qoCNFKYPCd1J3iXiWaQ4IKepNlQiYSl/ReV4DSc6jqCsYDSvWpG0YR3+M+NSlVhpuHtHebC3DhjIsFcOhLbZBxovLLgOwxKXVfQ6NI29gCqueASRiqb1jst5ZcCrMJc+RR3hxthi/ff2S+xFc5pJZ1TMPhCff7pOq87HyQuToH76QKq/gDf1KB8qsZCabSf1BdiQ+lkVRwYywHq+X1l14bIPm8kcc/e5ZHANxHU7CG1Hft+FEBBkqR8HJh2nZ+huvjqYd)

**Comment by J-Sek** [comment]
I was not able to reproduce it with Windows Narrator, but had some success with Orca. Screenreader is "stuck" the first time, but after 2-3 attempts it gets its shit together and up/down navigation works. Pretty annoying, but I have no clue what might be missing on our side.

**Comment by maxg-uiuc** [comment]
> I was not able to reproduce it with Windows Narrator, but had some success with Orca. Screenreader is "stuck" the first time, but after 2-3 attempts it gets its shit together and up/down navigation works. Pretty annoying, but I have no clue what might be missing on our side.

I went back to test again, and today it seemed to have no issues navigating the list. Genuinely not sure if this was a Vuetify thing now or just Narrator jank, but it's interesting that Orca got stuck for you. When I initially submitted, I also tested a standard HTML <select> and that worked fine. Really confusing.

**Comment by J-Sek** [comment]
Chrome auto-suggestions also screw things badly regardless of a tool and OS (forcing user to click `Escape` key before further interactions).
